src/pykeen/models/nbase.py

```diff
@@ -350,7 +348,6 @@ def __init__(
 
     def _build_representations(
         self,
-        triples_factory: KGInfo,
```

Comment:
iirc, this was because inductive models had to build representations for non-training triples factories. Has this been changed?

Reply:
I didn’t see this getting used anywhere else but I’ll double check

Reply:
you're right, reverted in 442ff38a